pkg/secrethub/http/options.go

```diff
@@ -7,7 +7,7 @@ import (
 
 type Option func(*Client)
 
-func WithRemote(remote string) Option {
+func WithServerURL(remote string) Option {
```

Comment:
Argument could be named `url` too, do we want to validate it, and/or do we want it to be a `url.URL` type?

Reply:
Missed that rename, fixed.

Using an `url.URL` makes it a lot less friendly to use this function. If someone uses a malformed URL, he'll find out at runtime. Sounds okay to me.

Reply:
Alright, other issue then is how do we return an error when the server cannot be reached? I think that error message looks really weird.

Reply:
That's a good point. I will create a separate PR for that when this is done. Will also include client timeouts in that.